mungegithub/mungers/flake-manager.go

```diff
@@ -127,6 +127,17 @@ func (p *FlakeManager) isIndividualFlake(f cache.Flake) bool {
 	return true
 }
 
+func (p *FlakeManager) listPreviousIssues(title string) []string {
+	if previousIssues := p.finder.AllIssuesForKey(title); len(previousIssues) > 0 {
```

Comment:
given how this is used the test for len() > 0 seems useless. You could just return the empty []string{} but this is fine too.